docs/topics/tour/kotlin-tour-functions.md

```diff
@@ -308,6 +350,9 @@ In the previous example:
 * `text` has type `String`.
 * the function returns the result of the [`.uppercase()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/uppercase.html)
 function called on `text`.
+* the entire lambda expression is assigned to the `upperCaseString` variable with the assignment operator `=`.
```

Comment:
I'm surprised to see you start list items almost everywhere with a lowercase letter. I don't think it's an option in the guide and it's not consistent with the rest of the documentation